src/nupic/support/__init__.py

```diff
@@ -431,13 +431,18 @@ def makeKey(name):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/var/run/syslog"'
   elif platform.startswith('linux'):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/dev/log"'
+  elif platform.startswith('win'):
+    replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"log"'
```

Comment:
@vitaly-krugl - thoughts on this? In general, this logging code is pretty hacky compared to what we have since implemented for use in numenta-apps.